src/Error/ExceptionRenderer.php

```diff
@@ -339,25 +340,16 @@ protected function _message(Throwable $exception, int $code): string
      * Get template for rendering exception info.
      *
      * @param \Throwable $exception Exception instance.
-     * @param string $method Method name.
+     * @param string $template Template name.
```

Comment:
This is confusing since the function is documented as getting the template name.

Reply:
I am hoping there is a way to combine the `_method()` and `_template()` functions because they're hard to use together. The names don't really make sense either.

Reply:
I don't mind reverting the argument name to what it was before.